### ISSUE TITLE
[3] Safari cannot restore wallet

### ISSUE THREAD

**Issue by gibbsfromncis** [bitshares/bitshares-ui#1125]
When I try to `restore account` from my local file nothing happens. 

Reproduce:
1) Open wallet.bitshares on Safari
2) Click `restore your account`
3) Select file
4) Button "Restore" or "Password Field" doesn't appear.

<img width="573" alt="screenshot 2018-02-07 14 50 04" src="https://user-images.githubusercontent.com/35899973/35919772-5513fb3a-0c16-11e8-8b53-b51d9e89b20b.png">

I understand that we do not support Safari (Actually I can understand if we do not support IE9 but why Safari? 😢 For the last year I didn't have any bad experience with it on React apps )

But sometimes we will start support all modern browsers so lets close our eyes for small bugs regarding design/ux but keep our project without critical issues like this one.

Please if you disagree just keep this bug in low priority. Don't close it.

**Comment by wmbutler** [comment]
We can support it if someone will go through and test it. Happy to add hours to this if you want to test every feature.

**Comment by gibbsfromncis** [comment]
@wmbutler Hm... Looks like interesting chance to become more familiar with Bitshares.  

Lets keep this bug like a separate issue. I'll open new issue with a list of features to test and will work on it on the upcoming Sprint.

**Comment by svk31** [comment]
The main reason Safari is not supported is that saving files via filesaver.js did not work properly, ref this issue. https://github.com/eligrey/FileSaver.js/issues/12.

It appears that issue has now been fixed, so we could update filesaver and possible start supporting Safari again.

**Comment by gibbsfromncis** [comment]
@svk31 thanks for your feedback. I'll work on it!

**Comment by startailcoon** [comment]
Do we have Safari support yet, and can we close this issue?

**Comment by svk31** [comment]
I don't have a Mac anymore so I can't test, if someone can install the latest version of filesaver and then try to download a backup bin file that would answer your question.

**Comment by wmbutler** [comment]
I just tested with Safari and it's not possible to restore from a backup.

**Comment by ppitonak** [comment]
There is the same problem with Firefox 61 on Fedora 28 so it's not Mac/Safari only.

**Comment by ppitonak** [comment]
When will this get to the staging env?

**Comment by wmbutler** [comment]
Not until 10/17.

**Comment by ppitonak** [comment]
Works in Firefox 62
